vars/managerPipeline.groovy

```diff
@@ -71,6 +71,10 @@ def call(Map pipelineParams) {
             string(defaultValue: "${pipelineParams.get('scylla_mgmt_pkg', '')}",
                    description: 'Url to the scylla manager packages',
                    name: 'scylla_mgmt_pkg')
+
+            string(defaultValue: "/dev/sda1",
```

Comment:
why do we need this to be changeable via pipeline params?

Reply:
as I've wrote in approving this, but that's  depends on the urgency this needs to be fixed:

LGTM

but I would recommand refacotring this, and have configuration yaml per platform.
and pass multiple yamls to the test_config param, instead of adding each crazy little diff as a pipeline param.

it would make pipeline much clearer

Reply:
or we could get it on the flight, instead of hard coding it, no?

Reply:
1) The param value should be set in YAML only, I don't see any reason to give an option to set it in the Job. It should be set once per OS type and version.
2) Looking at the name I understand that `debian9-manager-sanity.jenkinsfile` is a YAML for Debian9, @ShlomiBalalis / @fgelcer  is it used for other OS-s too? If so create separate YAMLs.

Reply:
> 1. The param value should be set in YAML only, I don't see any reason to give an option to set it in the Job. It should be set once per OS type and version.

So why not to do at python code? as we already run lots of different commands per OSes, this could be added.
and we could get this value from commands like `lsblk` or something like `ls /dev/*da`

> 2. Looking at the name I understand that `debian9-manager-sanity.jenkinsfile` is a YAML for Debian9, @ShlomiBalalis / @fgelcer  is it used for other OS-s too? If so create separate YAMLs.

Reply:
you can't get the value from OS command like `lsblk` since you need provide block device mappings to boto3 when the instance is created. There is a way to do it automatically, I created a [task](https://trello.com/c/lRuFPkg7/1727-auto-detect-root-block-device-for-aws-nodes) for it. But meanwhile@ShlomiBalalis remove the param from the Jenkins pipeline code